resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Wonders if {PRONOUN/m_c/poss} fur is unkempt and whether others can notice",
+            "Is busy sharpening {PRONOUN/m_c/poss} claws to look stronger",
+            "Worries that {PRONOUN/m_c/subject} {VERB/m_c/aren't/isn't} contributing enough",
+            "Thinks about all the mistakes {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} made"
+        ],
+        "main_trait_constraint": [
+            "insecure"
+        ]
+    },
+    {
+        "id": "lonesome_warrior",
+        "thoughts": [
+            "Is sitting alone",
+            "Looks on as others share tongues around the camp clearing",
+            "Is eating a piece of fresh-kill in a corner of camp",
+            "Wants to be alone...",
+            "Is sitting quietly on the far side of camp",
+            "Wonders if everyone laughs with {PRONOUN/m_c/object} or at {PRONOUN/m_c/object}...",
+            "Thinks that {PRONOUN/m_c/subject} {VERB/m_c/are/is} alone in {PRONOUN/m_c/poss} thoughts and beliefs",
+            "Is looking longingly at other cats as they talk amongst themselves",
+            "Is nowhere to be seen around camp",
+            "Just can't get away from others",
+            "Was surprised a cat wanted to speak to {PRONOUN/m_c/object}",
+            "Wonders if anyone would miss {PRONOUN/m_c/object} if {PRONOUN/m_c/subject} travelled far away",
+            "Is unsure of how to start up a conversation",
+            "Is fumbling with {PRONOUN/m_c/poss} words",
+            "Is enjoying some peace and quiet away from others",
+            "Is seeking out a place where {PRONOUN/m_c/subject} can be by {PRONOUN/m_c/self} for a bit",
+            "Feels bad about never being invited to share tongues",
+            "Spitefully swears that {PRONOUN/m_c/subject} {VERB/m_c/don't/doesn't} need others, {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} just fine!",
+            "Feels the need to escape camp, the presence of everyone else just unbearable today",
+            "Doesn't know why {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} always alone"
+        ],
+        "main_trait_constraint": [
+            "lonesome"
+        ]
+    },
+    {
+        "id": "lonesome_no_kit",
+        "thoughts": [
+            "Took a nap out in the territory away from others",
+            "Went out to be by {PRONOUN/m_c/self}",
+            "Is thinking wistfully about the past...",
+            "Went on a long, moonlit stroll the other night",
+            "Took a long walk all on {PRONOUN/m_c/poss} own"
+        ],
+        "main_trait_constraint": [
+            "lonesome"
+        ],
+        "main_status_constraint": [
+            "apprentice",
+            "medicine cat apprentice",
+            "mediator apprentice",
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+        ]   
+    },
+    {
+        "id": "loving_warrior",
```

Comment:
thoughts that mention another cat of specific status need to be in a thought group with that status required

Reply:
Changed warriors to clanmates